src/main/com/example009/radek/v3/FileService.java

```diff
@@ -0,0 +1,102 @@
+package main.com.example009.radek.v3;
+
+import java.io.*;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileService {
+
+    private final String PATH_FILE_RESULTS = "src/main/com/example009/radek/v3/calculator-results/";
```

Comment:
tady do tohoto mista tyto soubory nepaatri, mozna to nebylo uvedeno spravne v zadani, melo by to byt ve slozce main/resources ... a v tech resources vytvor example009/v3 kde to ulozis

Reply:
Hotovo